os/net/routing/rpl-classic/rpl-icmp6.c

```diff
@@ -657,7 +657,8 @@ dao_input_storing(void)
   uint8_t subopt_type;
   uip_ipaddr_t prefix;
   uip_ds6_route_t *rep;
-  uint8_t buffer_length;
+  uint16_t buffer_length;
```

Comment:
Could these two variables be declared on the lines where they are initialized instead?